shared/entities/cost-inputs/project-size.entity.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  BaseEntity,
+  Unique,
+  ManyToOne,
+  JoinColumn,
+} from "typeorm";
+import { Country } from "../country.entity";
+import { ECOSYSTEM } from "../ecosystem.enum";
+import { ACTIVITY } from "../activity.enum";
+
+@Entity("project_size")
+@Unique(["country", "ecosystem", "activity"])
+export class ProjectSize extends BaseEntity {
+  @PrimaryGeneratedColumn("uuid")
+  id: string;
+
+  @ManyToOne(() => Country, (country) => country.code, { onDelete: "CASCADE" })
+  @JoinColumn()
```

Comment:
Another important bit is that we need to set a name for the join column, otherwise it will created using the property's name, that is, camel case:
![image](https://github.com/user-attachments/assets/b2e2b1f0-c9cb-493b-bd9e-239ab954f6d1)
```suggestion
    @JoinColumn({ name: "country_code" })
```
This way I think we can also use value to set this property:

`project.countryCode = 'ESP'`

instead of

`project.country = { code: 'ESP }`

Minor detail, but the naming convention should be consistent I think